server/package.json

```diff
@@ -4,9 +4,9 @@
     "private": true,
     "scripts": {
         "test": "NODE_ENV=test ./node_modules/.bin/mocha --exit --timeout 30000",
-        "dev": "export NODE_ENV=development&&nodemon app.js",
-        "prod": "export NODE_ENV=production&&nodemon app.js",
-        "stage": "export NODE_ENV=staging&&nodemon app.js"
+        "dev": "set NODE_ENV=development&&nodemon app.js",
+        "prod": "set NODE_ENV=production&&nodemon app.js",
+        "stage": "set NODE_ENV=staging&&nodemon app.js"
```

Comment:
are we going to keep changing export and set? is there a way to make this easier for mac and window users?

Reply:
yes, I tried with one common module but it didn't worked, I will try some other options today.